### DISCUSSION TITLE
想要一个全是冷笑话的评论区

### DISCUSSION THREAD

**Discussion by Robin-txdy** [share121/inter-knot#4774] [灌水]
![IMG_20240208_142446](https://github.com/user-attachments/assets/12e5d5c5-ed08-4a05-a01c-bdeb973bb06f)
如题，欢迎大家踊跃评论！

**Comment by jidianzhuxing** [comment]
你知道怎么才能让蜜蜂多起来吗？很简单，只要话密(花蜜)就行了。

**Comment by lllliufeng** [comment]
飞霄的原名叫萨兰，她的母亲是不是可以叫做萨母（萨姆）:p

**Reply by lllliufeng** [reply]
你知道为什么赛诺离不开提瓦特吗？
因为赛诺不能离开原神，不然就会离神<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Reply by lllliufeng** [reply]
貘泽为什么会选咸鱼做武器？
因为大隐隐鱼（于）市<img src="https://bbs-static.miyoushe.com/static/2023/05/31/b8339a6e239cd6bc312890b5a2b1d2b4_5203775527809838507.png" alt="希儿 看不见我" width="75" >

**Reply by Bicjet** [reply]
原来可以发表情的吗？

**Reply by aomengxin** [reply]
> 你知道为什么赛诺离不开提瓦特吗？ 因为赛诺不能离开原神，不然就会离神<img alt="三月七 告退" width="75" src="https://camo.githubusercontent.com/281cebcce0eaff2f2b15cc06dc6b07eeb55bfd755872d487b3c9f8a7ffaae9ad/68747470733a2f2f6262732d7374617469632e6d69796f757368652e636f6d2f7374617469632f323032342f30372f33312f37333334366632623938643435623632646139313339643237373466613835365f383634313131343136353038383331353735342e706e67">

冷死了

**Comment by JiuXiat** [comment]
好冷

**Comment by Hui9527-oss** [comment]
好冷<img src="https://bbs-static.miyoushe.com/static/2024/03/21/8fe47df50ba7e17e012e8ba5baaaad49_4838310462379389617.png" alt="丽都漫步-安东" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/8fe47df50ba7e17e012e8ba5baaaad49_4838310462379389617.png" alt="丽都漫步-安东" width="75" >

**Comment by qingshuiglaze** [comment]
我在大学以前只认识一个姓于的，一来到沈阳就认识了仨，俩舍友加一个助导，这何尝不是一种于人众呢

**Comment by LinkboyTvT** [comment]
你们知道9月秋老虎有多热吗！你们知道你们讲的冷笑话有多低级吗！根本冷不下来！看来是时候拿出我压箱底的冷笑————化了……(TvT)

**Comment by fzdqz** [comment]
知道流萤为什么和开拓者玩石头剪刀布的时候从来不出剪刀吗？
因为流萤说过了要向你展示她的全部（拳布）￣﹁￣

**Comment by fuyutukikuze** [comment]
有一种两性花，雌蕊在授粉后柱头上会长出新芽，而雄蕊会慢慢凋零。一次，一只蜜蜂问雌蕊："授粉之后雄蕊就会凋零了，你会想念它吗？"雌蕊此时已经长出了芽苞，它回答道："芽已结，我不相思。"

**Comment by mztxy229** [comment]
没开玩笑体温瞬间下降了<img src="https://bbs-static.miyoushe.com/static/2023/05/22/a2fa805a0a782cdb8ab44ddaf8d5343d_9213146029750742963.png" alt="阿姬倒地了02" width="75" >

**Comment by Leoileoileoi** [comment]
有一天，A问B：“你知道为什么飞机飞这么高，不会碰到星星吗？”
B想了想说：“不知道。”
A淡定地说：“因为星星会‘闪’~”

**Comment by HLXS-HLXS** [comment]
好冷啊<img src="https://upload-bbs.miyoushe.com/upload/2022/12/08/a2fa805a0a782cdb8ab44ddaf8d5343d_7408497524870727211.png" alt="阿姬倒地" width="75" >

**Comment by WuxuRiver** [comment]
我晕，刚注册这个网站就看到这么冷的评论区，绳网(゜-゜)つロ干杯

**Comment by 596wjr** [comment]
![Uploading Screenshot_2024-09-14-00-31-59-933_com.miHoYo.hkrpg.jpg…]()

**Comment by Fluorine-z** [comment]
逛个绳网给我离神干出来了

**Comment by Kazuha-hhh** [comment]
一瓶可乐放在那不喝，那就是摆饰可乐

**Comment by nabeilishi** [comment]
赛诺，什么时候？！

**Comment by aino2333** [comment]
赛诺？何时来的？

**Comment by LeoneMafia** [comment]
为什么橘猫很容易发胖？
因为大橘为重

**Comment by Orangered81** [comment]
zzz没了谁才会变得斯文？

赛斯，因为梅（没）赛德（得）斯<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/c0ee663156c5bb4dc518341f2fd231dd_2752194756869066332.png" alt="驭空 叹气" width="75" >

**Reply by Orangered81** [reply]
## 为什么抽卡不能站在草上面？
# 应为芳草萋萋（七七）<img src="https://img-static.mihoyo.com/communityweb/upload/068a8b611fae7cde48470ebfe5e21847.png" alt="米游姬得意" width="75" >

**Comment by jiangwuchang** [comment]
太可怕了，全是赛诺

**Comment by muxia666** [comment]
你知道为什么妮露站在水上就动不了了吗，因为她变成了水泥路（妮露）

**Comment by dhczyg** [comment]
我们这气温直降二十度就是这栋楼干的罢（悲）

**Comment by hpp326** [comment]
天津理工大学的校长是不是可以被称作“天理的维系者”（x） ​

**Comment by Hjhnull** [comment]
为什么要把芙芙放在旁边，因为猫咪会说芙(服)了

**Comment by 7Hello12** [comment]
xxx

**Comment by KZ7-JPG** [comment]
如果你吃辣，那你不要第一个吃，因为first blood（不辣的）

**Comment by Zzy17391229** [comment]
你知道为什么心海看着那么轻，却是所有人中最重的吗？因为愚人众（鱼人重）

**Comment by friedericus1848** [comment]
莱特欧拉练拳记录时间为了方便会直接撕下日历计时，所以是莱欧撕历

**Comment by yuyuyubi** [comment]
你知道为什么一朵花好笑吗，因为花它有梗

**Comment by Aviaryd** [comment]
你知道什么动物最安静，大猩猩，因为他会敲咪咪（悄咪咪）

**Comment by sg15012291951** [comment]
犹太人跟面包有什么区别？
答：面包在烤箱里不会尖叫

**Reply by Aviaryd** [reply]
西海

**Comment by Loxia1121** [comment]
为什么那维莱特是美露辛和枫丹人们的光？因为大家都叫他“那位light大人”

**Comment by Grease63489** [comment]
好冷啊~~~

**Comment by hpp326** [comment]
历史课上，老师在提瓦特发明了蒸汽机

**Comment by GoodLeaf46** [comment]
赛诺闭嘴，闭嘴也闭嘴

**Reply by WWWIIINNN114** [reply]
来这里的可能不止有赛诺，也许还有大孙子和zzz的柳姐（

**Comment by Pattisland** [comment]
我也来一个：星见雅的文采其实很好，因为她是“高耳姬”。

**Comment by Grease63489** [comment]
你好，我现在无法亲自回复你的邮件。我会尽快给你回复。

**Comment by shaoq-256** [comment]
牛奶和可乐在一起聊天，聊着聊着可乐不说话了
因为可乐没气儿了
：D

**Comment by shaoq-256** [comment]
还有一个虽然不冷，但是很地狱：
-黑人的什么最白？
-主人

**Comment by wAi-0713** [comment]
知道镜子碎成什么样子最重吗？
圆形，因为破镜重圆

**Comment by WWWIIINNN114** [comment]
知道芙卡洛斯去哪里最安全吗，答案是高速公路
因为高速公路禁止掉头

**Comment by Dchoclate** [comment]
都说人们会看见一件事中的利和弊，有些人只看见了弊，可利呢？
可利（可莉）在原神里
有些人只看见了利，可弊呢？
可弊（科比）坠机了

**Comment by Grease63489** [comment]
你好，我现在无法亲自回复你的邮件。我会尽快给你回复。

**Comment by l-wenyu-I** [comment]
刚来就让我刷到这个帖子，真是个巧合，就像……
欸嘿;-)

**Comment by Ezhou9** [comment]
₍˄·͈༝·͈˄*₎◞ ̑̑

**Comment by Dchoclate** [comment]
都说人们会看见一件事中的利和弊，但有些人只看见了利，可弊呢？可弊（科比）坠机了

**Comment by BSJ3456A** [comment]
好人和坏人坐同一辆公交车 哪个人会先下车呢？
——是坏人 因为好人做（坐）到底/ᐠ - ˕ -マ Ⳋ

**Reply by BSJ3456A** [reply]
你知道中世纪为什么要烧s女巫吗？
因为这样牧师就可以得到许多switch(witch翻译成中文为女巫)

**Reply by BSJ3456A** [reply]
秦始皇到了现在最适合从事哪种职业？
——特种兵 因为他自古就是反恐（孔）精英啊

**Comment by kind-style** [comment]
Which kind of bee can't fly?
